lib/good_job/active_job_extensions/concurrency.rb

```diff
@@ -97,7 +97,7 @@ def good_job_concurrency_key
       # @return [Object] concurrency key
       def _good_job_concurrency_key
         key = self.class.good_job_concurrency_config[:key]
-        return if key.blank?
+        return "#{self.class.name}-#{queue_name}-#{ActiveJob::Arguments.serialize(arguments)}" if key.blank?
```

Comment:
> I think the default should only be triggered if `!key?(:key)` so that it can be overridden with `nil` in a subclass to disable

Done + added a test for that.

> I think it also might be good to extract that to its own method, just in case someone wants to more easily monkeypatch it.

That sounds like a nice idea, done.